apps/verticalhome/js/more_apps_navigation.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+(function (exports) {
+
+  var MoreAppsNavigation = {
+    controls: null,
+    selectedElemIndex: 0,
+    selectElement: null,
+
+    init: function () {
+      this.controls = app.grid.getItems();
+      this.reset();
+    },
+
+    reset: function () {
+      if (this.selectElement) {
+        this.selectElement.classList.remove('selected');
+      }
+      this.selectedElemIndex = 0;
+      this.selectElement = this.controls[this.selectedElemIndex].element;
+      this.selectElement.classList.add('selected');
+      this.clearNavigationEvents();
+      window.addEventListener('keydown', this);
```

Comment:
This should be in init(), not reset(), I think.  When you close the more apps, I assume that you're calling reset, right? So you don't want an active event listener in that case, right?